pages/form/returning-members.js

```diff
@@ -72,22 +77,15 @@ const SUBTEAMS = [
   'Team Hub',
 ];
 
-const ACTIVE_STATUS = [
+const NEXT_TERM_ACTIVITY = [
   'Yes, I will continue on the team, and I will be on campus (or working locally)',
   'Yes, I will continue on the team remotely and can come to campus if needed/possible',
   'Yes, I will continue on the team remotely only',
   'Undecided or unsure',
   'No, taking the term off',
 ];
 
-const MEMBERSHIP = [
-  'Member',
-  'Exec/lead/advisor',
-  'Coop',
-  'Not active on Waterloop this term',
-];
-
-const CONTINUING_STATUS = [
+const NEXT_TERM_ROLE = [
   'Continue with my sub-team',
   'Transfer to another sub-team (please specify)',
```

Comment:
This option asks to specify, but there's no way to specify. This would require the creation of a connected radio button + text input... can we make this?

Reply:
Nope. Not going to do that. I'll just remove the (please specify) option.